packages/web/toast/package.json

```
@@ -0,0 +1,4 @@
+{
+  "main": "./index.js",
+  "types": "../dist/flash/index.d.ts"
```

```suggestion
  "types": "../dist/toast/index.d.ts"
```

This should be toast instead of flash now, right? Now that we have src/toast/index.ts